### PR TITLE
Localize some Treat Wounds macro strings

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

**Comment by stwlam** [comment]
Get the rest while you're in there

**Comment by simonward** [comment]
> Get the rest while you're in there

The only other notification in this file currently uses game.i18n.format(). It's not clear from the header docs that NotificationOptions.localize goes through format(), or just through localize().

**Comment by stwlam** [comment]
Just localize.

File: static/lang/action-en.json [hunk 1]
```diff
@@ -692,6 +692,7 @@
                     "RiskySurgery": "Risky Surgery"
                 },
                 "Label": "Treat Wounds",
+                "MedicineDC": "Medicine DC:",
```
Comment (stwlam) [inline static/lang/action-en.json:695]: use "PF2E.InlineCheck.DCWithName"

Comment (simonward) [reply static/lang/action-en.json:695]: If we do, we won't get the colon on the end.

PF2E.Actions.TreatWounds.MedicineDC:
Screen Shot 2023-06-20 at 4 22 08 PM

PF2E.InlineCheck.DCWithName:
Screen Shot 2023-06-20 at 4 24 47 PM

Comment (stwlam) [reply static/lang/action-en.json:695]: Then remove the others.

**Pull request merged**